src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -90,6 +90,7 @@
 
     <link rel="stylesheet" href="css/main.css">
     <link rel="stylesheet" href="css/print.css">
+    <link rel="stylesheet" href="css/app-print.css">
```

Comment:
I'm not sure if I know what's wrong with that screen capture, but I think that since the node is appended to a specifically named element, `export-print-preview-map`, which is explicitly named in the CSS file as the parent, then the rules can be fairly precisely applied. The reason I opted for that was that there seemed to be inconsistency with applied styles when the DOM node was moved around, or when events acted upon it.